# Empty goal
bool use_goal # if false, will keep current values
bool use_estimates
bool use_nullspace
int32 rod_arm
int32 surface_arm
float64 nullspace_gain
float64 mechanism_timeout
int32 num_of_experiments
std_msgs/String bag_prefix
float64 vd_amp
float64 wd_amp
float64 vd_freq
float64 wd_freq
float64 force
bool randomize_desired_state
float64 x_min
float64 x_max
float64 theta_min
float64 theta_max
float64 f_min
float64 f_max

int32 LEFT = 0
int32 RIGHT = 1
---
# Empty result
---
# Provide status of what's being run as feedback
string current_action
string progress
